api/src/opentrons/protocol_engine/state/labware.py

```diff
@@ -47,6 +51,8 @@
     "opentrons/usascientific_96_wellplate_2.4ml_deep/1",
 }
 
+_PROBE_DECK_SLOT_OFFSET = DeckSlotOffsetVector(x=10, y=0, z=3)
```

Comment:
Same comment, there's no need to create a new class for this. I might move the "magic slot" up here, too.

```suggestion
_PIPETTE_PROBE_ATTACH_SLOT = DeckSlotName.SLOT_5
_PIPETTE_PROBE_ATTACH_X_OFFSET_FROM_SLOT = 10.0
_PIPETTE_PROBE_ATTACH_Z_OFFSET_FROM_SLOT = 3.0

_PIPETTE_ATTACH_SLOT = DeckSlotName.SLOT_2
```